admin_scripts/ubuntu/ubuntu_16.04/layout/remove_new_release_message.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+
+release_upgrades_file=/etc/update-manager/release-upgrades
+
+# Simple backup
+if [ ! -f $release_upgrades_file.org ]
+then
+    cp $release_upgrades_file $release_upgrades_file.org
+fi
+
+# Replace Prompt with never value
+sed -i 's/Prompt=.*/Prompt=never/' $release_upgrades_file
```

Comment:
Would this be better handled by uninstalling `ubuntu-release-upgrader-*`? I think that's what we did when we most recently built the image...